task2/src/domen/users/user.router.js

```diff
@@ -25,21 +26,19 @@ import userDatabaseMethods from '../../../repository.js'
 
       })
 
-      userRouter.post('/', (request, response) => {
-            const body = request.body
-
-            if(!Object.keys(request.body).length) {
+      userRouter.post('/', validator, (request, response) => {
```

Comment:
I shouldn't send isDelete when creating a user. It is a field which works behind the scenes. Think about sending "activated: true" when creating a github account. It should be false when creating a user and false when deleting a user, and also be handled by developer. Not by requestor.